app/controllers/v0/form210779_controller.rb

```diff
@@ -9,28 +9,50 @@ class Form210779Controller < ApplicationController
     skip_before_action :authenticate
 
     def create
-      confirmation_number = SecureRandom.uuid
-      submitted_at = Time.current
-
-      render json: {
-        data: {
-          id: '12345',
-          type: 'saved_claims',
-          attributes: {
-            submitted_at: submitted_at.iso8601,
-            regional_office: [],
-            confirmation_number:,
-            guid: confirmation_number,
-            form: '21-0779'
-          }
-        }
-      }, status: :ok
+      params.require(:form)
+
+      claim = SavedClaim::Form210779.new(form: params[:form].to_json)
+
+      if claim.save
+        # claim.process_attachments!
+
+        Rails.logger.info(
+          "ClaimID=#{claim.confirmation_number} Form=#{claim.class::FORM}"
+        )
+        StatsD.increment("#{stats_key}.success")
+
+        render json: SavedClaimSerializer.new(claim)
+      else
+        StatsD.increment("#{stats_key}.failure")
+        raise Common::Exceptions::ValidationErrors, claim
+      end
+    rescue => e
+      # Include validation errors when present; helpful in logs/Sentry.
+      Rails.logger.error(
+        'Form210779: error submitting claim',
+        { error: e.message, claim_errors: defined?(claim) && claim&.errors&.full_messages }
+      )
```

Comment:
Will the message contain PII?

Reply:
No, if there's validation errors the message will be something like this:
["/veteraninformation/fullname/first string length at `/veteranInformation/fullName/first` is greater than: 12"]